pypesto/objective/util.py

```diff
@@ -13,7 +13,7 @@ def res_to_chi2(res: np.ndarray):
     """
     if res is None:
         return None
-    return 0.5 * np.power(res, 2).sum()
+    return np.dot(res, res)
```

Comment:
why remove the 0.5? I think I've seen both implementations in use.

Reply:
We now use the chi2 value provided by AMICI, which doesn't have the 0.5 factor. I would actually prefer to have the 0.5 factor here but find it more important to be consistent with what amici returns

Reply:
will add comment

Reply:
Should be fine I guess (not aware of having used this factor anywhere explicitly, except maybe in a test. Please also update the docstring above.

Reply:
yep, thats what I did :)

Reply:
ok I was too late. all done already :)